drivers/spi/spi_ll_stm32.c

```diff
@@ -823,6 +823,17 @@ static const struct spi_driver_api api_funcs = {
 	.release = spi_stm32_release,
 };
 
+static inline bool spi_stm32_is_subghzspi(const struct device *dev)
+{
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32_spi_subghz)
+	const struct spi_stm32_config *cfg = dev->config;
+
+	return cfg->use_subghzspi_nss;
+#endif
+
+	return false;
```

Comment:
I would put this in an #else case. The case where this results in two unconditional return does not feel quite right.

Reply:
Agreed.